CIP-0006/README.md

```diff
@@ -1,20 +1,25 @@
 ---
 CIP: 6
 Title: Stake Pool Extended Metadata
-Authors: Markus Gufler <gufmar@gmail.com>, Mike Fullman <mike@fullman.net>
-Comments-URI: https://github.com/cardano-foundation/CIPs/pull/15
-Status: Draft
-Type: Standards
+Status: Active
+Category: Tools
+Authors:
+  - Markus Gufler <gufmar@gmail.com>
+  - Mike Fullman <mike@fullman.net>
+Implementors:
+  - pooltool.io
+  - cexplorer.io
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/15
 Created: 2020-07-20
-Updated: 2020-11-24 2nd key-pair for validation 2021-02-08 json schema
 License: CC-BY-4.0
 ---
 
 ## Abstract
 
 This CIP defines the concept of extended metadata for pools that is referenced from the pool registration data stored on chain.
 
-## Motivation
+## Motivation: why is this CIP necessary?
 
 As the ecosystem around Cardano stake pools proliferate so will the desire to slice, organize and search pool information dynamically.  Currently the metadata referenced on chain provides 512 bytes that can be allocated across the four information categories ([delegation-design-specification Section 4.2)](https://github.com/input-output-hk/cardano-ledger/releases/latest/download/shelley-delegation.pdf):
```

Comment:
```suggestion
As the ecosystem around Cardano stake pools proliferate so will the desire to slice, organize and search pool information dynamically.  Currently the metadata referenced on chain provides 512 bytes that can be allocated across the four information categories ([delegation-design-specification Section 4.2)](https://github.com/IntersectMBO/cardano-ledger/releases/latest/download/shelley-delegation.pdf):
```
do we want to do this?

Reply:
@Ryun1 @Crypto2099 @KtorZ two possible ways of answering your question (although definitely **yes** we want to correct the out of date links):

- We want to change these links to point to IntersectMBO because the IOHK links have become non-canonical and eventually the redirects will stop working (or get lost behind further redirects someday).
- We want to use the versions in our own repos since the academic releases of these technical papers are generally paywalled or at least require academic accounts to view them.